Final polish on Lambda Function URL (#2042)
SumoLogic/sumologic-documentation: docs/alerts/webhook-connections/aws-lambda.md

Have your webhook URL handy by copying and pasting it to a notepad. You'll need it to configure your webhook connection in the next section.
:::

> To authenticate SumoLogic backend and allow webhook calls you need to create in your AWS Account an IAM User account with very restricted priviledges. Creating Lambda Function URL you need to create your own policy while building an API Gateway you can use AWS managed policy. In two following sections you can find detailed steps for both cases. 

Reviewer: ```suggestion
To authenticate the Sumo Logic Logic backend and allow webhook calls, you'll need to create in your AWS Account an IAM User account with very restricted privileges. Creating Lambda Function URL you need to create your own policy while building an API Gateway you can use AWS managed policy. In the following sections, you'll can find detailed steps for both cases. 
```

### Lambda Function URL

If you use the Lambda Function URL to expose a Lambda function, you'll need to:

1. Create **Customer managed** policy with the **lambda:InvokeFunctionUrl** action allowed.
  :::note
  This is different from the **lambda:InvokeFunction** action, which is used in the AWS-managed IAM Policy **AWSLambdaRole**.
  :::

>   

Reviewer: ```suggestion

```

       ]
   }
   ```

>    
> 2. Create an IAM user account and assign **Customer managed** policy created in step 1.
> 

Reviewer: ```suggestion
2. Create an IAM user account and assign **Customer managed** policy created in step 1.
```
